ocaml/testsuite/tests/tool-expect-test/include.ml

```diff
@@ -0,0 +1,12 @@
+(* TEST
+ include unix;
```

Comment:
```suggestion
 include unix;
 hasunix;
```
This directive makes sure that tests that require unix only run on unix environments. (We'll probably want this before upstreaming.)